ruma-events/tests/redacted.rs

```diff
@@ -108,7 +134,7 @@ fn redacted_aliases_deserialize() {
         AnyRoomEventStub::RedactedState(AnyRedactedStateEventStub::RoomAliases(RedactedStateEventStub {
             event_id, content, ..
```

Comment:
This should really be `content: RedactedRoomAliases { ... }` rather than the ZST check below.